src/K9OCRS/ClientApp/src/App.js

```diff
@@ -6,6 +6,9 @@ import { FetchData } from './components/FetchData';
 import { Counter } from './components/Counter';
 import ClassTypesDashboard from './pages/ClassTypesDashboard';
 import ClassTypeSetup from './pages/ClassTypeSetup';
+import DogRegistration from './pages/DogRegistration';
+import DogProfile from './pages/DogProfile';
+import DogList from './pages/DogList'
```

Comment:
You'll need to update these changes for the comment below so the code doesn't break. I would also suggest changing the name of the folders/components to reflect these changes and make the names consistent with the Figma design.

you can just click "Add suggestion to batch" for both comments and then commit them in one go 👍 
```suggestion
import DogSetup from './pages/DogRegistration';
import DogDetails from './pages/DogProfile';
import MyDogs from './pages/DogList'
```